### PR TITLE
feat: Added Dockerfile for ubuntu image

### PR DESCRIPTION
**Pull request by frgfm**
This PR introduces the following modifications:
- adds a Dockerfile for a basic ubuntu image
- updated README
- added CI job to check that the docker builds successfully

Any feedback is welcome!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM python:3.8.1-slim
+
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE 1
```
Comment (chloeskt) [inline Dockerfile:4]: Thank you @frgfm for this PR ! 

Small question: what is the use of this env variable in our case ?

Comment (frgfm) [reply Dockerfile:4]: It prevents the generation of `.pyc` upon compilation :)

Comment (chloeskt) [reply Dockerfile:4]: Yes, but why should we add this ? only to save disk space ?

Comment (frgfm) [reply Dockerfile:4]: yes, and simply to keep the number of files to a minimum. It's a very common thing to add in a dockerfile

Comment (chloeskt) [reply Dockerfile:4]: Ok thanks ! 😄

**Review by chloeskt — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -60,9 +60,9 @@ pip install pyronear
 
 ## Usage
 
-### datasets
+### Python package
 
-Access all pyrovision datasets just like any `torchvision.datasets.VisionDataset`:
+You can use the library like any other python package to analyze your documents as follows:
```
Comment (MateoLostanlen) [inline README.md:65]: Why did you removed this here ?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -110,3 +110,13 @@ jobs:
...
+
+  docker-ready:
+    runs-on: ubuntu-latest
+    needs: pkg-install
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build docker image
+        run: docker build . -t pyrovision-py3.8.1-torch1.7-slim
+      - name: Run docker container
+        run: docker run pyrovision-py3.8.1-torch1.7-slim python -c 'import pyrovision'
```
Comment (MateoLostanlen) [inline .github/workflows/main.yml]: 'import pyrovision' gives an error on my machine (SyntaxError: EOL while scanning string literal), I need to use "import pyrovision" instead

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -110,3 +110,13 @@ jobs:
...
         run: |
           sphinx-build docs/source docs/build -a
+
+  docker-ready:
+    runs-on: ubuntu-latest
+    needs: pkg-install
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build docker image
+        run: docker build . -t pyrovision-py3.8.1-torch1.7-slim
```
Comment (MateoLostanlen) [inline .github/workflows/main.yml:120]: The bild work well, thanks for that :)

File: README.md (hunk 2 above)
Comment (frgfm) [reply README.md:65]: I was starting to think that this is the purpose of the documentation, not the README. Usage instructions should be plain user-friendly examples of how to use the package in my opinion.

File: .github/workflows/main.yml (hunk 3 above)
Comment (frgfm) [reply .github/workflows/main.yml]: Oh good to know, it means that this behaves differently on windows then! I'll fix this right away

File: README.md (hunk 2 above)
Comment (MateoLostanlen) [reply README.md:65]: ok ok

**Review by MateoLostanlen — APPROVED** [review]

**Comment by MateoLostanlen** [comment]
All good :)

**Pull request merged**